File: auto_click/src/btpy/src/btpy/btpy_maths/mod/vector_sum/vector_sum.py




def vector_sum(array1:list[int], 
        array2:list[int])->list[int]:
    """
    Función que suma los elementos 
    de dos arrays de números en la 
    misma posición del array.
    """
    #
    new_array = array1.copy()
    for i in range(len(array1)):
        new_array[i] += array2[i]
    return new_array

File: btjs_page/to_work/old_btjs/math/mod/vector_sum.js


/*
Función que suma los elementos 
de dos arrays de números en la 
misma posición del array.
*/
export function vector_sum(ARRAY_1, 
            ARRAY_2) {
    let new_array = [...ARRAY_1];
    for (let i = 0; i < ARRAY_1.length; i++) {
      new_array[i] += ARRAY_2[i];
    }
    return new_array;
}